fusor-ember-cli/app/models/deployment-plan.js

```diff
@@ -1,8 +1,63 @@
+import Ember from 'ember';
 import DS from 'ember-data';
 
 export default DS.Model.extend({
     name: DS.attr('string'),
     uuid: DS.attr('string'),
     roles: DS.hasMany('deployment-role', {inverse: null, async: false}),
-    parameters: DS.hasMany('deployment-plan-parameter', {inverse: null, async: false})
+    parameters: DS.hasMany('deployment-plan-parameter', {inverse: null, async: false}),
+
+    externalNetworkInterface: Ember.computed('parameters.@each.value', {
```

Comment:
@cfchase, this looks good to user computed properties on the model to retrieve `getParamValue`